package.json

```diff
@@ -12,8 +12,8 @@
   "files": [
     "build-lib"
   ],
-  "main": "build-lib/umd/index.umd.js",
-  "module": "build-lib/es/index.es.js",
+  "main": "build-lib/umd/production/index.min.js",
+  "module": "build-lib/es/production/index.min.js",
```

Comment:
Do we need to minify `es` code?

Reply:
I haven't seen this before, but I could also not have looked very far.  My sense has been that you do not need to minify this since it is used for development and then minified for the web when necessary.

Reply:
Good question, I am not sure. I figured that if someone would be using the ES modules in the browser, directly from `unpkg` for example (without their own minification step), then producing a minified file may be helpful. For reference, the un-minified ES index.js file is 19.5 MB, while the minified index.min.js file is 3.4 MB. Minified gets down below 1 MB with gzipping.

Reply:
<img width="949" alt="Screen Shot 2020-04-21 at 11 49 26 AM" src="https://user-images.githubusercontent.com/7525285/79891639-48bc7280-83cf-11ea-829b-eeabf0fc962b.png">

Reply:
That makes sense.  Looks good then!